src/include/storage/store/table_statistics.h

```diff
@@ -14,17 +15,31 @@ namespace storage {
 using lock_t = std::unique_lock<std::mutex>;
 using atomic_uint64_vec_t = std::vector<std::atomic<uint64_t>>;
 
-class TableStatistics {
+using TablePropertyStats =
```

Comment:
Usually we give complicated types an alias to simplify its usage, such as nested maps, vectors, etc. But I think this map is pretty straightforward, and I would just use it as it is.

Reply:
Yes, it's just long and I had to add it to a bunch of different parameter lists where it's mostly just being passed around.